test/absinthe/adapters/language_conventions_test.exs

```diff
@@ -11,6 +11,16 @@ defmodule Absinthe.Adapter.LanguageConventionsTest do
     test "converts external camelcase variable names to underscore" do
       assert "foo_bar" = LanguageConventions.to_internal_name("fooBar", :variable)
     end
+
+    test "preserves external operation names" do
+      assert "fooBar" = LanguageConventions.to_internal_name("fooBar", :operation)
```

Comment:
Does this imply a change in behavior? Seems like previously an operation name would be transformed?

I imagine this isn't a breaking change in the usual sense, but I suspect some APIs might track the operation name for monitoring, etc and could be relying on the existing behavior - say you have a dashboard tracking API usage based on operation name...

Reply:
moving this comment to the relevant function